src/_main_/utils/base_model.py

```diff
@@ -25,3 +21,21 @@ def to_json(self, full=False, tiny_info=False):
 
     class Meta:
         abstract = True
+
+class BaseModel(RootModel):
```

Comment:
i'm a bit concerned about what happens to migrations if we do this.  Does it cause some extra migration files to show up?

Reply:
No It doesn't create any extra migration files